lib/schema.js

```diff
@@ -672,14 +355,14 @@ class Metaschema {
   //   args - <Object>, action arguments
   // Returns: <MetaschemaError> | <null>
   validateAction(category, actionName, args) {
-    const categoryActions = this.actions.get(category);
+    const categoryActions = this.categories.get(category);
```

Comment:
It's not `categoryActions`, it's `category` itself here. We can either change the variable name or extract the actions here:
```suggestion
    const categoryActions = this.categories.get(category).actions;
```